_preload.lua

```diff
@@ -224,6 +224,12 @@
 		tokens = true,
 	}
 	
+	api.register {
+		name = "androidapplibname",
```

Comment:
I'd go for applibname as a bare minimum as it errs on the side of being descriptive; my reaction to 'libname' was "isn't that (or something very similar) already used in premake?".

Note that VS' macro (only available at the Ant packaging stage, i.e. not in Project Properties) is $(AndroidAppLibName); AndroidManifest.xml is referencing an "android.app.libName" in the metadata, etc.